onnxruntime/python/tools/quantization/README.md

```diff
@@ -101,4 +101,26 @@ If False, the inputs/activations are quantized using dynamic scale and zero poin
             {
                 'resnet_model/Relu_3:0': [np.int8(0), np.float32(0.011359662748873234)],
                 'resnet_model/Relu_4:0': [np.uint8(0), np.float32(0.011359662748873234)]
-            }
\ No newline at end of file
+            }
+
+## Calibration
```

Comment:
Add this above quantization section because calibration comes before quantization so logically in the readme also this flow makes more sense

Reply:
Done!